cmd/wallet-web/src/router/index.js

```diff
@@ -17,9 +17,7 @@ const router = createRouter({
 router.beforeEach(async (to, from, next) => {
   store.dispatch('agent/flushStore');
   if (to.path === '/gnap') {
-    // TODO Issue-1720 create keypair only once if not found
-    const gnapKeyPair = await createKeyPair();
-    store.dispatch('updateGnapKeyPair', gnapKeyPair);
+    const gnapKeyPair = await getGnapKeyPair();
```

Comment:
vuex store call shouldn't be inside mixin javascript logic,

Reply:
This one is in index.js not in mixin package

Reply:
I was asking to bring it back to `index.js` from `await createKeyPair()`